commit 92be69bd962e0539bcaf8b3d13e92810fba0c3c0 (refs/changes/68/2968/5)
Author: Jan Wyszynski <jan.wyszynski@joyent.com>
Date:   2017-12-07T21:56:07+00:00 (1 year, 10 months ago)
    
    MANTA-3491 muskie startup emits error message to stderr
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/lib/uploads/create.js b/lib/uploads/create.js
index 805116d..f034135 100644
--- a/lib/uploads/create.js
+++ b/lib/uploads/create.js
@@ -18,7 +18,13 @@ var util = require('util');
 var verror = require('verror');
 var Ajv = require('ajv');
 var ajv = new Ajv({allErrors: true});
-require('ajv-keywords')(ajv);
+
+/*
+ * All keywords provided by ajv-keywords must be explicitly imported here. Any
+ * ajv-keywords used without being imported will be silently ignored during
+ * validation.
+ */
+require('ajv-keywords')(ajv, ['prohibited']);
 
 var auth = require('../auth');
 var common = require('../common');
